scripts/scil_compute_maps_for_particle_filter_tracking.py

```diff
@@ -64,39 +64,39 @@ def main():
     if args.verbose:
         logging.basicConfig(level=logging.DEBUG)
 
-    assert_inputs_exist(parser, [args.wm, args.gm, args.csf])
+    assert_inputs_exist(parser, [args.in_wm, args.in_gm, args.in_csf])
     assert_outputs_exist(parser, args,
                          [args.include, args.exclude, args.interface])
 
-    wm_pve = nib.load(args.wm)
-    logging.info('"{0}" loaded as WM PVE map.'.format(args.wm))
-    gm_pve = nib.load(args.gm)
-    logging.info('"{0}" loaded as GM PVE map.'.format(args.gm))
-    csf_pve = nib.load(args.csf)
-    logging.info('"{0}" loaded as CSF PVE map.'.format(args.csf))
+    wm_pve = nib.load(args.in_wm)
+    logging.info('"{0}" loaded as WM PVE map.'.format(args.in_wm))
+    gm_pve = nib.load(args.in_gm)
+    logging.info('"{0}" loaded as GM PVE map.'.format(args.in_gm))
+    csf_pve = nib.load(args.in_csf)
+    logging.info('"{0}" loaded as CSF PVE map.'.format(args.in_csf))
 
     background = np.ones(gm_pve.shape)
-    background[gm_pve.get_data() > 0] = 0
-    background[wm_pve.get_data() > 0] = 0
-    background[csf_pve.get_data() > 0] = 0
+    background[gm_pve.get_fdata(dtype=np.float32) > 0] = 0
+    background[wm_pve.get_fdata(dtype=np.float32) > 0] = 0
+    background[csf_pve.get_fdata(dtype=np.float32) > 0] = 0
 
-    include_map = gm_pve.get_data()
+    include_map = gm_pve.get_fdata(dtype=np.float32)
     include_map[background > 0] = 1
 
-    exclude_map = csf_pve.get_data()
+    exclude_map = csf_pve.get_fdata(dtype=np.float32)
 
     interface = np.zeros(gm_pve.shape)
-    interface[gm_pve.get_data() >= args.int_thres] = 1
-    interface[wm_pve.get_data() < args.int_thres] = 0
+    interface[gm_pve.get_fdata(dtype=np.float32) >= args.int_thres] = 1
+    interface[wm_pve.get_fdata(dtype=np.float32) < args.int_thres] = 0
 
     logging.info('The interface "{0}" contains {1} voxels.'.format(
         args.interface, int(np.sum(interface))))
 
-    nib.Nifti1Image(include_map.astype('float32'),
+    nib.Nifti1Image(include_map.astype(np.float32),
```

Comment:
In scilpy, the vast majority of saving is done using nib.save(nib.Nifti1Image()). I think you should change it just for consistency.